forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -22,10 +23,19 @@ import java.util.*
 
 internal data class ProxyRequestObject(val pin: TextElement, val card_number_token: String)
 
+internal data class POSProxyRequestObject(
```

Comment:
Can we rename this to something "refund" specific. We'll also be sending POS-specific stuff for balance checks